source/tests/system/nirfmxgsm_driver_api_tests.cpp

```diff
@@ -0,0 +1,82 @@
+#include <gtest/gtest.h>
+
+#include "device_server.h"
+#include "niRFmxGSM.h"
+#include "nirfmxgsm/nirfmxgsm_client.h"
+
+using namespace ::testing;
+using namespace nirfmxgsm_grpc;
+namespace client = nirfmxgsm_grpc::experimental::client;
+
+namespace ni {
+namespace tests {
+namespace system {
+namespace {
+
+const auto PXI_5663E = "5663E";
+
+template <typename TResponse>
+void EXPECT_SUCCESS(const TResponse& response)
+{
+  EXPECT_EQ(0, response.status());
+}
+
+class NiRFmxGSMDriverApiTests : public Test {
+ protected:
+  NiRFmxGSMDriverApiTests()
+      : device_server_(DeviceServerInterface::Singleton()),
+        stub_(NiRFmxGSM::NewStub(device_server_->InProcessChannel()))
+  {
+    device_server_->ResetServer();
+  }
+
+  virtual ~NiRFmxGSMDriverApiTests() {}
+
+  void TearDown() override
+  {
+    device_server_->ResetServer();
+  }
+
+  const std::unique_ptr<NiRFmxGSM::Stub>& stub() const
+  {
+    return stub_;
+  }
+
+  template <typename TResponse>
+  void EXPECT_SUCCESS(const nidevice_grpc::Session& session, const TResponse& response)
```

Comment:
This might be more helpful later as more tests are added, but there's some Macros in "rfmx_macros.h" for EXPECT_SUCCESS and related helpers you might want to re-use going forward instead of this.

Reply:
I based this file on an older version of nirfmxlte_driver_api_tests.cpp, so I'll update it.

Reply:
Updated in commit 5df2e1418bf5d0c607d07e769eae02c2aa604918